src/main/java/core/basesyntax/service/impl/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ReportGenerator;
+import java.util.stream.Collectors;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+    @Override
+    public String getReport() {
+        return "fruit,quantity" + Storage.reports.entrySet().stream()
```

Comment:
The `getReport` method directly concatenates strings which can lead to performance issues if the dataset is large. Consider using `StringBuilder` for more efficient string concatenation in loops.